config/initializers/assets.rb

```diff
@@ -9,8 +9,3 @@
 # Add additional assets to the asset load path
 # Rails.application.config.assets.paths << Emoji.images_path
 
-# Precompile additional assets (application.js.coffee.erb, application.css, and all non-JS/CSS are already added)
-Rails.application.config.assets.precompile += %w( diagram.js graphing.js map_marker.js ace.js )
```

Comment:
I guess that must be what's happening, and application.js.coffee does it's own bundling of `components/*` and `pages/*`.